Scripts/odbcoracle-table-definition.sql

```diff
@@ -16,9 +16,9 @@ CREATE TABLE VWBPLIBF.VTLI0036C(
 	V_ORA_RAWDATA Raw(50) NULL,
 	V_ORA_DATA Varchar2(50) NULL,
 	V_ORA_CODE Number(18, 0) NULL,
-	V_ORA_XML Xmltype NULL,
```

Comment:
Why have you removed XML? I said to comment out the test, in the code, not remove it from the table. Otherwise that column should be treated just like all the other columns.

Reply:
I know all that. You told me already.

It needs to be there. A fix may be applied in the future. Customers have XML columns and inserts to the file need to work, even if the XML is not updated.

Reply:
I tried to set *SQLNULL and update the xml column but, I was unable to proceed on further if the column read-only on xml field.

Strange thing is we will not be able to see any data in the table and shows no records, but select list and insert operation iterate once.

I tried to check what is read only column in Lansa docs I count find any such,  Also there is no example of Blob field has readonly access flag.

Reply:
> I tried to set *SQLNULL and update the xml column but, I was unable to proceed on further if the column read-only on xml field.
> 
> Strange thing is we will not be able to see any data in the table and shows no records, but select list and insert operation iterate once.
> 
> I tried to check what is read only column in Lansa docs I count find any such, Also there is no example of Blob field has readonly access flag.

Is there an error on the second insert?

I don't care about what is in the XML field. Only that the insert/update of all the other fields work.

Reply:
There was no error  , but when we update some column we will  have an issue and it fails.
And we wont be able to see any data in the data preview.

You want me to add them now ?

Reply:
> There was no error , but when we update some column we will have an issue and it fails. And we wont be able to see any data in the data preview.
> 
> You want me to add them now ?

What issue? What fails? Details.
Update fails when XML field is in the table? When its provided on the UPDATE command? What?

Reply:
If all other fields update, then do it. If not, tell me the error.

Reply:
Yes update fails when XML field is in the table. same code works when the blob field is removed. I suspect read-only access might be the issue.

Reply:
I understand we need to remove XML column for Oracle test as it has a defect, if this is the case code is up-to date with all comments resolved. Let me know if any points to be taken care.

Reply:
Can we resolve this conversation or anything else is expected to resolve on this point.